code/datums/jobs/squads.dm

```diff
@@ -366,6 +367,11 @@
 	for(var/mob/living/marine AS in marines_list)
 		marine.playsound_local(marine, sound, 35)
 		marine.play_screen_text("<span class='maptext' style=font-size:24pt;text-align:center valign='top'><u>[header]</u></span><br>" + message, message_type, message_color)
+	if(sender?.voice && SStts.tts_enabled)
+		var/list/extra_filters = list(TTS_FILTER_RADIO)
+		if(isrobot(sender))
+			extra_filters += TTS_FILTER_SILICON
+		INVOKE_ASYNC(SStts, TYPE_PROC_REF(/datum/controller/subsystem/tts, queue_tts_message), marines_list, treated_message["tts_message"], sender.get_default_language(), sender.voice, sender.voice_filter, local = TRUE, pitch = sender.pitch, special_filters = extra_filters.Join("|"), directionality = FALSE)
```

Comment:
marines_list should be the listeners arg, first arg is 'target' which seems to be the actual speaker. Same with the xeno annoucements